src/app/api.py

```diff
@@ -2,19 +2,32 @@
 Core FastAPI app (setup, middleware)
 """
 import logging
+import os
 import time
 from datetime import datetime
+from typing import Dict, List, Optional
 
 import sentry_sdk
 import uvicorn  # type: ignore
-from fastapi import FastAPI, Request
+from fastapi import Body, Depends, FastAPI, Request
+from fastapi.responses import HTMLResponse, JSONResponse
 from fastapi.staticfiles import StaticFiles
+from fastapi.templating import Jinja2Templates
 from sentry_sdk.integrations.asgi import SentryAsgiMiddleware
 
+from src.app import configuration
 from src.app.environment import get_settings
 from src.app.log import configure_logging
 from src.app.monitor import api_router as monitor_router
-from src.jbi.router import api_router as jbi_router
+from src.jbi.bugzilla import BugzillaWebhookRequest
+from src.jbi.models import Actions
+from src.jbi.runner import IgnoreInvalidRequestError, execute_action
+from src.jbi.services import get_jira
+
+SRC_DIR = os.path.dirname(os.path.dirname(os.path.realpath(__file__)))
```

Comment:
It might me nice to switch to the `Pathlib` api here
```python
from pathlib import Path
# ...
SRC_DIR = Path(__file__).parents[1]
# ...
templates = Jinja2Templates(directory=SRC_DIR / "templates")
# ...
app.mount("/static", StaticFiles(directory=SRC_DIR / "static"), name="static")
```